src/main/java/redis/clients/jedis/util/Version.java

```diff
@@ -0,0 +1,40 @@
+package redis.clients.jedis.util;
+
+import java.util.Arrays;
+
+public class Version implements Comparable<Version> {
```

Comment:
The name is **_generic_** version. `5.2.0-beta5` is a valid Jedis version. This class does not handle that.

Reply:
Intention here is to cover the basically what we need here instead of introducing a whole third party library and add a dependency. May be we can rename it just to emphasize it is not handling various formats and needs. Would you feel more comfortable that way ?

Reply:
Yes.

RedisVersion like would be safer.

I think we can even package private this class in `redis.clients.jedis` package.

Reply:
done